mass_update_incidents/mass_update_incidents.py

```diff
@@ -9,11 +11,9 @@
 
 import pdpyras
 
-# If you are trying to update over 10,000 incidents at a time, use the `since`,`until`, and `time_zone`
```

Comment:
Since you are removing this from the comments, can you please add this limitation to the readme?

Reply:
Done, thank you!